docs/source/getting-started/tracking-server-overview/index.rst

```diff
@@ -0,0 +1,327 @@
+  2023/10/25 22:59:38 INFO mlflow.utils.credentials: Succesfully signed in Databricks!
+
+
+Connect MLflow Session to Databricks CE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We have set up the credentials, now we need to tell MLflow to send the data into Databricks CE. 
+To do so, we will use :py:func:`mlflow.set_tracking_uri("databricks")` to port MLflow to Databricks CE. Basically 
+it is the command below. Please note that you need to always use `"databricks"` as the keyword.
+
+.. code-block:: python
+
+  mlflow.set_tracking_uri("databricks")
+
+Now you are ready to go! Let's try starting an MLflow experiment and log some dummy metrics and view it in our UI.
+
+.. code-block:: python
+
+  mlflow.set_experiment("/check-databricks-connection")
+
+  with mlflow.start_run():
+      mlflow.log_metric("foo", 1)
+      mlflow.log_metric("bar", 2)
+
+.. code-block:: text
+
+  2023/10/25 23:15:42 INFO mlflow.tracking.fluent: Experiment with name '/check-databricks-ce-connection' does not exist. Creating a new experiment.
+
+
+View Your Experiment on Databricks CE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now let's navigate to Databricks CE to view the experiment result. Log in to your 
+`Databricks CE <https://community.cloud.databricks.com/>`_ account, and click on top left to select machine learning 
+in the drop down list. Then click on the experiment icon. See the screenshot below:
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/databricks-ce-landing-page.png 
+    :width: 800px
+    :align: center
+    :alt: Landing page of Databricks MLflow server
+
+
+In the "Experiments" view, you should be able to find the experiment "/check-databricks-ce-connection", similar to 
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/databricks-ce-experiment-view.png
+    :width: 800px
+    :align: center
+    :alt: Experiment view of Databricks MLflow server
+
+Clicking on the run name, in our example it is "youthful-lamb-287" (it's a randomly generated name, you will see 
+a different name in your CE console), will bring you to the run view, similar to 
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/databricks-ce-run-view.png
+    :width: 800px
+    :align: center
+    :alt: Experiment view of Databricks MLflow server
+
+In the run view, you will see our dummy metrics `"foo"` and `"bar"` are logged successfully.
+
+Conclusion
+^^^^^^^^^^^
+
+That's all about how to use Databricks CE as the tracking server. Please see the pros and cons
+of this method below:
+
+* **Pros**
+  
+  * Effortless setup.
+  * Free. 
+  * Good for collaboration, e.g., you can share your MLflow experiment with your teammates easily.
+  * Compatible for developing on cloud-based notebook, e.g., Google Colab.
+  * Compatible for developing on cloud VM.
+
+* **Cons**
+  
+  * Has quota limit of experiments/runs.
+  * No model registration/serving support.
+
+Method 3: Use Production Databricks Workspace
```

Comment:
Can we make this provider agnostic? SaaS MLflow Workspace and give an exempli gratia list inclusive of Databricks, AzureML

Reply:
done